base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
Same as the situation for Dict, SortedDict, and OrderedDict. Some sets might be sorted, others not.